Add note about omise module

ptmkenny opened the pull request:
I discovered a payment gateway not on this list, so I added it.  I also clarified the opening sentence so that it is clear that this list should only contain modules that have not been abandoned.

bojanz commented:
```
 -Here is a running document of known and supported payment gateways.
 +Here is a list of known payment gateways for which support is currently being provided.
```
So we went from "supported" to "for which support is currently being provided". Doesn't that say the same thing? The wording also makes it seem like we're the ones providing support, so we'd need to clarify that it's the maintainers who are providing the support. I'd rather go with "Here is a list of known payment gateways, supported by their maintainers"

ptmkenny commented:
@bojanz It's not the same thing because "supported gateways" is ambiguous.  It could mean "compatible gateways" or it could mean "gateways that currently have support from the maintainers."

I rewrote it to clarify this, but as you noted, it should also be made clear that the support comes from the module maintainers.  I've updated the PR.

bojanz commented:
Thanks!
